pkg/sitewise/datasource.go

```diff
@@ -134,7 +159,7 @@ func (ds *Datasource) HealthCheck(ctx context.Context, req *backend.CheckHealthR
 }
 
 func (ds *Datasource) HandleInterpolatedPropertyValueQuery(ctx context.Context, _ *backend.QueryDataRequest, query *models.AssetPropertyValueQuery) (data.Frames, error) {
-	sw, err := ds.GetClient(ctx, query.BaseQuery.AwsRegion)
+	sw, err := ds.getClient(ctx, query.BaseQuery.AwsRegion)
```

Comment:
Fixing this https://github.com/grafana/iot-sitewise-datasource/pull/402#discussion_r2021448316